test/sql/test_materialized_view/R/test_materialized_view_rewrite

```diff
@@ -0,0 +1,365 @@
+-- name: test_materialized_view_rewrite
+create table user_tags (time date, user_id int, user_name varchar(20), tag_id int) partition by range (time)  (partition p1 values less than MAXVALUE) distributed by hash(time) buckets 3 properties('replication_num' = '1');
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 1, 'a', 1);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 1, 'b', 2);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 1, 'c', 3);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 1, 'd', 4);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 1, 'e', 5);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 2, 'e', 5);
+-- result:
+-- !result
+insert into user_tags values('2023-04-13', 3, 'e', 6);
+-- result:
+-- !result
+create materialized view user_tags_mv1  distributed by hash(user_id) as select user_id, bitmap_union(to_bitmap(tag_id)) from user_tags group by user_id order by user_id;
+-- result:
+-- !result
+select sleep(2);
+-- result:
+1
+-- !result
+set enable_materialized_view_rewrite = off;
+-- result:
+-- !result
+select user_id, count(distinct tag_id) from user_tags group by user_id order by user_id;
+-- result:
+1	5
+2	1
+3	1
+-- !result
+select user_id, bitmap_union_count(to_bitmap(tag_id)) from user_tags group by user_id order by user_id;
+-- result:
+1	5
+2	1
+3	1
+-- !result
+select user_id, bitmap_count(bitmap_union(to_bitmap(tag_id))) x from user_tags group by user_id order by user_id;
+-- result:
+1	5
+2	1
+3	1
+-- !result
+set enable_materialized_view_rewrite = on;
```

Comment:
this statement only turn on/off the rewrite, but without verify whether rewrite or not ?

Reply:
each agg type:
- collect result for rewrite is off;
- test no-rollup rewrite result;
- test rollup rewrite result.

NOTE: `New Tester` can not output `explain` result, just sleep 2s to ensure mv is ready.


![image](https://user-images.githubusercontent.com/5104975/234197084-aa7f8655-3610-4df3-b627-ee22dc8b9ec6.png)